src/user/projects/popups/create-project.js

```diff
@@ -0,0 +1,179 @@
+import React, { Component } from "react";
+import { Modal, Button, Row, Col, Form } from "react-bootstrap";
+import "./popups.scss";
+
+export class CreateProject extends Component {
+  // eslint-disable-next-line
```

Comment:
enable linting and fix if any linting error.

Reply:
@devesh-verma told me so remove the create-project feature so I,just removed it